### PR TITLE
chore: bring in llvm toolchain so we can update //js/private/test/image snapshots on darwin

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: e2e/npm_link_package-esm/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,6 @@
+###############################################################################
```
Comment (jbedard) [inline e2e/npm_link_package-esm/MODULE.bazel:1]: Delete...

Comment (gregmagolan) [reply e2e/npm_link_package-esm/MODULE.bazel:1]: ✅

**Comment by gregmagolan** [comment]
Re-based and targeted `main` for this one. I'll consolidate the duplicate http_archives before landing.

File: e2e/npm_link_package-esm/MODULE.bazel (hunk 1 above)
Comment (jbedard) [reply e2e/npm_link_package-esm/MODULE.bazel:1]: It's still there...

**Review by jbedard — APPROVED** [review]

File: e2e/npm_link_package-esm/MODULE.bazel (hunk 1 above)
Comment (gregmagolan) [reply e2e/npm_link_package-esm/MODULE.bazel:1]: 🤦

Comment (gregmagolan) [reply e2e/npm_link_package-esm/MODULE.bazel:1]: huh... git commit says they are removed
```$ gd HEAD~ --name-only
.bazelrc
WORKSPACE
WORKSPACE.bzlmod
js/dev_repositories.bzl
```
I think the GH UI is being buggy here

Comment (gregmagolan) [reply e2e/npm_link_package-esm/MODULE.bazel]: There we go. Pushed a few more times and it updated correctly.

**Review by jbedard — APPROVED** [review]

**Pull request merged**